contracts/examples/mintable-erc1155-transfer/FxMintableERC1155RootTunnel.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import {ERC1155} from "../../lib/ERC1155.sol";
+import {FxERC1155} from "../../tokens/FxERC1155.sol";
```

Comment:
Good catch, but interface `IFxERC1155` can be used instead of `FxERC1155`. Will save deployment cost.

Reply:
7880b5ac74d1471b4f2170566f8afed7131aed86